src/lib/stitching/gravity/stitching.ts

```diff
@@ -89,21 +84,42 @@ export const gravityStitchingEnvironment = (
             }
           `,
           resolve: ({ startAt: _startAt, endAt: _endAt }) => {
+            moment.updateLocale("en", {
+              relativeTime: {
+                s: "%d second",
+                ss: "%d seconds",
+                m: "%d minute",
+                mm: "%d minutes",
+                h: "%d hour",
+                hh: "%d hours",
+                d: "%d day",
+                dd: "%d days",
+                M: "%d month",
+                MM: "%d months",
+                y: "%d year",
+                yy: "%d years",
+              },
+            })
```

Comment:
@pvinis - the reason this approach wasn't taken was because this (seemingly static / global) change on moment has already happened somewhere in auctions code (if you search for `updateLocale`). Not sure how to get around that other than tracking down the various call-sites and making sure things are still ok. That said, I could be misunderstanding moment!

Reply:
I thought the same thing after reading the docs, but I saw we have tests for the auctions part here https://github.com/artsy/metaphysics/blob/master/src/schema/v2/sale/__tests__/index.test.js#L597 for example, and they seem to pass.

So I'm not sure if the docs are strange and we both misunderstood, or if there is something else going on here. 🤔